pkg/cloud/config.go

```diff
@@ -0,0 +1,188 @@
+	// +optional	`
+	Password string `yaml:"password,omitempty"`
+}
+
+type Datacenter struct {
+	// CertificateAuthorityPath is the path to a cert file for the certificate authority.
+	// +optional
+	CertificateAuthorityPath string `yaml:"certificateAuthorityPath,omitempty"`
+	// CertificateAuthorityData contains PEM-encoded certificate authority certificates. Overrides CertificateAuthority.
+	// +optional
+	CertificateAuthorityData []byte `yaml:"certificateAuthorityData,omitempty"`
+	// Server is the initial contact point of the Scylla cluster.
+	// Example: https://hostname:port
+	Server string `yaml:"server"`
+	// TLSServerName is used to check server certificates. If TLSServerName is empty, the hostname used to contact the server is used.
+	// +optional
+	TLSServerName string `yaml:"tlsServerName,omitempty"`
+	// NodeDomain the domain suffix that is concatenated with host_id of the node driver wants to connect to.
+	// Example: host_id.<nodeDomain>
+	NodeDomain string `yaml:"nodeDomain"`
+	// InsecureSkipTLSVerify skips the validity check for the server's certificate. This will make your HTTPS connections insecure.
+	// +optional
+	InsecureSkipTLSVerify bool `yaml:"insecureSkipTlsVerify,omitempty"`
+	// ProxyURL is the URL to the proxy to be used for all requests made by this
+	// client. URLs with "http", "https", and "socks5" schemes are supported. If
+	// this configuration is not provided or the empty string, the client
+	// attempts to construct a proxy configuration from http_proxy and
+	// https_proxy environment variables. If these environment variables are not
+	// set, the client does not attempt to proxy requests.
+	// +optional
+	ProxyURL string `yaml:"proxyUrl,omitempty"`
+}
+
+type Context struct {
+	// DatacenterName is the name of the datacenter for this context.
+	DatacenterName string `yaml:"datacenterName"`
+	// AuthInfoName is the name of the authInfo for this context.
+	AuthInfoName string `yaml:"authInfoName"`
+}
+
+type Parameters struct {
+	// DefaultConsistency is the default consistency level used for queries.
+	// +optional
+	DefaultConsistency ConsistencyString `yaml:"defaultConsistency,omitempty"`
+}
+
+type ConsistencyString string
+
+// just AnyConsistency etc is better, but there's already SerialConsistency defined elsewhere.
+const (
+	DefaultAnyConsistency         ConsistencyString = "ANY"
+	DefaultOneConsistency         ConsistencyString = "ONE"
+	DefaultTwoConsistency         ConsistencyString = "TWO"
+	DefaultThreeConsistency       ConsistencyString = "THREE"
+	DefaultQuorumConsistency      ConsistencyString = "QUORUM"
+	DefaultAllConsistency         ConsistencyString = "ALL"
+	DefaultLocalQuorumConsistency ConsistencyString = "LOCAL_QUORUM"
+	DefaultEachQuorumConsistency  ConsistencyString = "EACH_QUORUM"
+	DefaultSerialConsistency      ConsistencyString = "SERIAL"
+	DefaultLocalSerialConsistency ConsistencyString = "LOCAL_SERIAL"
+	DefaultLocalOneConsistency    ConsistencyString = "LOCAL_ONE"
+)
```

Comment:
Why not use normal consistency settings in the driver?
Also SerialCosistency and Consistency are distinct things.

Reply:
Added second consistency to config and validation